service/handlers.go

```diff
@@ -44,10 +45,23 @@ func (rh *RouteHandler) ConstructPayload(w http.ResponseWriter, r *http.Request)
 		return 400, fmt.Errorf("Driver of type string not provided")
 	}
 	webhookRequestData["projectId"] = accountID
+	webhookRequestData["uuid"] = uuid.New()
 	jwt, err := util.CreateTokenWithPayload(webhookRequestData, PrivateKey)
 	if err != nil {
 		return 500, err
 	}
+	apiClient, err := rh.rcf.GetClient(accountID)
+	if err != nil {
+		return 500, err
+	}
+	name, ok := webhookRequestData["name"].(string)
+	if !ok {
+		return 400, fmt.Errorf("Name not provided")
+	}
+	err = saveWebhook(webhookRequestData["uuid"].(string), name, apiClient)
```

Comment:
dont get uuid from the map. Where you add it to the map, instead assign it to a variable and then use the variable here.